proposals/112-car-v2.md

```diff
@@ -0,0 +1,52 @@
+# CAR v2
+
+Authors: @willscott
+
+## What is the problem this project solves?
+_Describe the status quo, including any relevant context on the problem you're seeing that this project should solve. Who is the user you're solving for, and why do they care about this problem? Wherever possible, include pain points or problems that you've seen users experience to help motivate why solving this problem works towards top-line objectives._ 
+
+This project implements a second version of the Content addressed data ARchive format.
+The goal is to provide a standardized way to take a collection of content addressed data, as currently exists in the car format, and be able to efficiently provide random access reads and appends to it.
+We know through the carbs and carbon experiments that both of these are possible.
+
+With this functionality we can avoid the need to take an existing car file and import it back into a blockstore like badger before being able to perform random access reads over it.
+
+## Impact
+_What goals/OKRs are being addressed (for w3dt, a specific program, etc.)? Why is this project important? What do we get with this project that we can't get without it?_
+
+This project aims to impact the goals of:
+- low latency retrieval from filecoin
+
+These are core goals that the Bedrock program aims to address. 
+
+## The idea
+_Describe the proposed project solution, at a very high level. Stay at the level of the high-level requirements. Diagrams and interface descriptions can be useful, if you have any that help clarify and explain the idea._
+
+As described in the [sharded dag store](https://docs.google.com/document/d/118fJdf8HGK8tHYOCQfMVURW9gTF738D0weX-hbG-BvY/edit?ts=60bf961f) design document we want a minimal format for carv2 consisting of
```

Comment:
doc is private to PL, I don't think there's a good reason not to turn on public read-only (or even comment) access for this is there?

Reply:
we can update to point to the spec you're working on once it's in linkable form. I think there's enough of a description in the text here that anyone wandering by can get a reasonable sense of what this project work looks like, and if they really care about the internals they're of course welcome to reach out and ask us :)